manifests/m/MaxDiesel/UnrealCommander/3.5.7.1497/MaxDiesel.UnrealCommander.installer.yaml

```diff
@@ -0,0 +1,21 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: MaxDiesel.UnrealCommander
+PackageVersion: 3.5.7.1497
+Installers:
+- Architecture: neutral
+  InstallerType: inno
+  InstallerUrl: https://unrealcommander.org/download/evolution/uncomsetup3.57(build1497).exe
+  InstallerSha256: AF7CB526BEDE31F3E0D0FE478C38FCB76C20D81D7899CC43DE8E83769E01E548
+  InstallModes:
+  - interactive
+  - silent
+  - silentWithProgress
+  InstallerSwitches:
+    Silent: /VERYSILENT
+    SilentWithProgress: /SILENT

```

Comment:
```suggestion
```
By default, WinGet runs Inno installers with `/SILENT` so I believe this isn't needed.

Some Inno installers requires `/VERYSILENT` to suppress some installation prompts that couldn't be suppressed with `/SILENT`.

If the above is the case, then feel free to re-add it.